atom/browser/ui/file_dialog.cc

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2017 Brave Authors. All rights reserved.
```

Comment:
can we use file_select_helper instead or would that be a lot more work?

Reply:
it will requires more extra work including whole API interfaces changes. Can we do it as follow-up?

Reply:
the api for browser-laptop or just internal muon?

Reply:
I think it is both, they all use the same file_dialog interfaces now